lib/tasks/publishing_api.rake

```diff
@@ -32,6 +32,64 @@ namespace :publishing_api do
       ))
     end
   end
+
+  desc "Send publishable item links to Publishing API"
+  task publishing_api_patch_links: :environment do
+    $stdout.puts "# Sending all items' links to Publishing API"
+    $stdout.puts "# Ruby code is output for failed operations\n"
+
+    def patch_links(item)
+      retries = 0
+
+      begin
+        content_id = item.content_id
+        links = PublishingApiPresenters.presenter_for(item).links
+        if links
+          Whitehall.publishing_api_v2_client.patch_links(content_id, {links: links})
+        else
+          $stdout.puts "# No links for item with content_id: #{content_id}"
+        end
+      rescue GdsApi::TimedOutException, Timeout::Error
+        retries += 1
+        if retries <= 3
+          $stderr.puts "# Timeout: retry #{retries}"
+          sleep 0.5
+          retry
+        end
+        raise
+      end
+    rescue => err
+      message = "# Class: #{item.class}; id: #{item.id}; FAILURE: #{err.message}"[0..99]
+      $stderr.puts message
+      $stderr.puts("#" * message.length)
+      payload = { links: links } if links
+      $stderr.puts "Whitehall.publishing_api_v2_client.patch_links('#{content_id}', #{payload})\n" if links
+    end
+
+    [
+      Unpublishing,
+      AboutPage,
+      Classification,
+      MinisterialRole,
+      Organisation,
+      Person,
+      PolicyGroup,
+      TakePartPage,
+      WorldLocation,
+      StatisticsAnnouncement,
+    ].each do |klass|
+      count = klass.all.count
+      $stdout.puts "# Sending #{count} #{klass} items to Publishing API\n"
+      klass.find_each do |publishable_item|
+        patch_links(publishable_item)
+      end
+
+      $stdout.puts "# Sending published editions to Publishing API\n"
```

Comment:
This looks correct to me. The above `find_each` is for non-Editions, and this loop is for Editions. However, it it looks like this might be better served with 
```ruby
Edition.published.find_each do |edition|
```
then we don't need the guard condition